src/mutations/membership.ts

```diff
@@ -198,10 +198,12 @@ export default (queryConfig: QueryClientConfig) => {
                 },
                 queryConfig,
               );
-            // set map key to email
-            Object.values(membershipsResult.data).forEach((m) => {
-              dataForMemberships.data[m.member.email] = m;
-            });
+            // FIX: this is commented out as we will not receive the email from the backend anymore.
+            // we should remove this code anyways and move the whole process to the backend.
+            // // set map key to email
+            // Object.values(membershipsResult.data).forEach((m) => {
+            //   dataForMemberships.data[m.account.email] = m;
+            // });
```

Comment:
Should we create and issue and remove the comment for that ?

Reply:
I think @pyphilia is working on adding the necessary code in the backend ?

Reply:
I'm going to.